<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySimIIGui::Initiativess</class>
 <widget class="QWidget" name="MySimIIGui::Initiativess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>40000</width>
    <height>80000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Initiative settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QTabWidget" name="TabWid_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30000</width>
         <height>30000</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="TabContent1">
        <attribute name="title">
         <string>CFD</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_Tab1">
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEdit_Pressure"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_VelZ"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_VelY">
           <property name="text">
            <string>Vel.Y(m/s)：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_VelZ">
           <property name="text">
            <string>Vel.Z(m/s)：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_Pressure">
           <property name="text">
            <string>Pressure(N/m^2)：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_VelX"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_VelX">
           <property name="text">
            <string>Vel.X(m/s)：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_VelY"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent2">
        <attribute name="title">
         <string>Heat</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_Tab2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Temperature"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_Temperature">
           <property name="text">
            <string>Temperature(^oC)：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent3">
        <attribute name="title">
         <string>EM</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_Tab3">
         <item row="8" column="0">
          <widget class="QLabel" name="label_IncWaveY">
           <property name="text">
            <string>E.Y：</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_E_Z">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLineEdit" name="lineEdit_M_Z">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_IncPropX">
           <property name="text">
            <string>M.X：</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_IncPropZ">
           <property name="text">
            <string>M.Z：</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_E_Y">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="lineEdit_M_Y">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_E_X">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="lineEdit_M_X">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_IncWaveX">
           <property name="text">
            <string>E.X：</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_IncWaveZ">
           <property name="text">
            <string>E.Z：</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_IncPropY">
           <property name="text">
            <string>M.Y：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent5">
        <attribute name="title">
         <string>Elastic</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="7" column="1">
          <widget class="QLineEdit" name="LE_MeshUpdate2"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>MeshUpdate 3</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Displacement 1</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>MeshUpdate 1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Force 1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="LE_Displacement2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Force 3</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="LE_Displacement3"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Displacement 3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Displacement 2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LE_Force1"/>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="LE_MeshUpdate3"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="LE_Force2"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>MeshUpdate 2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="LE_Displacement1"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="LE_MeshUpdate1"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Force 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="LE_Force3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent4">
        <attribute name="title">
         <string>Self-Defining</string>
        </attribute>
        <widget class="QLabel" name="Labal_GeometryInfo">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Variable list (separated by &quot;,&quot;):</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit_SelfDef1">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>28</y>
           <width>311</width>
           <height>71</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30000</width>
           <height>30000</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Ex1: Time, Coordinate 1, Coordinate 2, Temperature&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text" stdset="0">
          <string>## Geometry Info ##</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_Blank2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>130</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="Labal_SelfDefinition2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>110</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>MATC function statements (separated by &quot;;&quot;):</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit_SelfDef2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>311</width>
           <height>81</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30000</width>
           <height>30000</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ex1: a=1.0; b=2.1; Tx(0)+a*Tx(1)-Tx(2)*Tx(3)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text" stdset="0">
          <string>## including as Elmer MATC function.</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Subdomains:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="tw_Subdomains"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
